leftwm/src/bin/leftwm.rs

```diff
@@ -149,6 +158,8 @@ fn start_leftwm() {
             }
         }
 
+        no_error_occured = evaluate_session(&mut leftwm_session) != SessionStatus::Error;
```

Comment:
You have a double negative logic here. Wouldn't it be nicer to read with:
```suggestion
        error_occured = evaluate_session(&mut leftwm_session) == SessionStatus::Error;
```